### PR TITLE
Docusaurus starter guide

### PR DESCRIPTION
**Pull request by Tate-CC**
This PR adds a getting started guide for Docusaurus.

### CODE REVIEW

File: guides/docusaurus-starter-guide/index.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  category: Starter Guides
+---
+
+Docusaurus is a content-based, JavaScript static-site generator with a focus on building optimized documentation websites.
+Docusaurus leverages Markdown and MDX to let you focus on writing your content while still allowing you to easily
+customize and extend your site with the power of React. By default, Docusaurus provides features for documentation sites, but
+it can be used to create any kind of site and supports advanced features like versioning, i18n, and search.
+
+CloudCannon makes it easy to store your content in your Git Repository and have non-developers update it.
+There’s no need to teach people about front matter, markdown or Git.
```
Comment (rycoll) [inline guides/docusaurus-starter-guide/index.mdx:25]:
```suggestion
There’s no need to teach people about front matter, Markdown or Git.
```

File: guides/docusaurus-starter-guide/configuring.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Some SSGs, like Jekyll, Hugo, and Eleventy, have strong opinions about how to organize the content in your repo.
+CloudCannon can run automatic processes on these SSGs and provide basic editing out of the box.
+Docusaurus is less opinionated and has a number of plugins which can change where and how content is stored.
+This means that for Docusaurus, CloudCannon out of the box only has the source editor available.
+
+### Editing terminology
+
+CloudCannon has a few concepts that make organizing data easier:
+
+- **Data**\: These are standalone data files. This is ideal for site configuration or files that don't need to repeat. Data is edited by altering to a single file.
```
Comment (rycoll) [inline guides/docusaurus-starter-guide/configuring.mdx:25]:
```suggestion
- **Data**\: These are standalone data files. This is ideal for site configuration or files that don't need to repeat. Data is edited by altering a single file.
```

File: guides/docusaurus-starter-guide/configuring.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+- Edit any data file
+
+You will have numerous editing options available to you:
+
+- Source Editor: edit your code directly within CloudCannon in a familiar-looking editor.
+- Data Editor: Edit your data and front matter in a recursive form full of customizable inputs.
+  - Add, remove, clone, rearrange and edit arrays
+  - Open objects and edit content to any depth
+  - Change data with date, color, number, URL, and more inputs
+- Content Editor: Edit your markdown files and data at the same time. A big content block for your markdown and an embedded data editor on the side. This is perfect for a combination of markdown and front matter files.
```
Comment (rycoll) [inline guides/docusaurus-starter-guide/configuring.mdx:60]:
```suggestion
- Content Editor: Edit your Markdown files and data at the same time. A big content block for your Markdown and an embedded data editor on the side. This is perfect for a combination of Markdown and front matter files.
```

File: guides/docusaurus-starter-guide/configuring.mdx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+The global configuration file is one of many places that configuration can exist. Some options can exist only in the global config. Others options can be configured in a number of places; these options always start with an `_` character.
+These options cascade like CSS: the more specific the options are, the more they take precedence. From least important to most important, the order is:
+
+1. Global configuration.
+2. Collection configuration.
+3. Front matter.
+4. Containing structure.
+
+### Configuring inputs
```
Comment (rycoll) [inline guides/docusaurus-starter-guide/configuring.mdx:81]: A couple links to docs about _inputs and _structures could be helpful here!

File: guides/docusaurus-starter-guide/rich-content-editing.mdx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+touching the underlying code.
+
+<comp.DocsImage
+  path="/static/docusaurus-starter-guide/content-editor.png"
+  alt="Screenshot of CloudCannon's content editor showing structured Snippets alongside Markdown content"
+  type="screenshot"
+/>
+
+To get started with Snippets in Docusaurus, you can follow this guide on setting up the builtin
+[MDX Component](/documentation/articles/editing-with-mdx-components/) and [Docusaurus](/documentation/articles/editing-with-docusaurus/) Snippets.
```
Comment (rycoll) [inline guides/docusaurus-starter-guide/rich-content-editing.mdx:29]: The "editing-with-docusaurus" link doesn't seem to go anywhere

**Review by rycoll — CHANGES_REQUESTED** [review]
I've got some small changes suggested, of varying levels of importance, but looks good on the whole!

File: guides/docusaurus-starter-guide/rich-content-editing.mdx (hunk 5 above)
Comment (Tate-CC) [reply guides/docusaurus-starter-guide/rich-content-editing.mdx:29]: This just needed a pull from main to get the new docusaurus snippets docs

File: guides/docusaurus-starter-guide/configuring.mdx (hunk 4 above)
Comment (Tate-CC) [reply guides/docusaurus-starter-guide/configuring.mdx:81]: Done!

**Review by rycoll — APPROVED** [review]
Nice, looks good to me!

**Pull request merged**